install_gh.sh

```diff
@@ -1,10 +1,10 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 set -ex
 
 GH_ARCH="amd64"
 
-VERSION=`curl --retry 12 --retry-delay 30 "https://api.github.com/repos/cli/cli/releases/latest" | grep '"tag_name"' | sed -E 's/.*"([^"]+)".*/\1/' | cut -c2-`
+VERSION=$(curl --retry 12 --retry-delay 30 "https://api.github.com/repos/cli/cli/releases/latest" | grep '"tag_name"' | sed -E 's/.*"([^"]+)".*/\1/' | cut -c2-)
```

Comment:
Can you add a space after `$(`, to have `$( curl...- )`

Reply:
Yes, and I'll make this consistent in the places that already had no spaces.